doc/code_snippets/snippets/mysql/instances.enabled/config.yaml

```diff
@@ -0,0 +1,25 @@
+app:
+  file: 'example.lua'
+
+groups:
+  group001:
+    replicasets:
+      replicaset001:
+        instances:
+          instance001:
+            iproto:
+              listen: "3301"
+            memtx:
+              memory: 128 * 1024 * 1024
+              min_tuple_size: 16
+              max_tuple_size: 128 * 1024 * 1024
+            wal:
+              mode: "none"
+              wal_max_size: 56 * 1024 * 1024
+            snapshot:
+              by_interval: 60 * 60
```

Comment:
These values are numbers, but here we have strings with expressions. It'll not be accepted.